src/components/POLine/Cost/CostForm.js

```diff
@@ -117,7 +124,16 @@ class CostForm extends Component {
           />
         </Col>
         <Col xs={6}>
-          <FieldCurrency />
```

Comment:
Could you please state a point of moving code from a separate small file to a common bigger file? IMHO, module system is more convenient in development. What do you think?

Reply:
As we've discussed in <FieldCurrency> main functionality was a translation for currencies, but now it's skipped, and my solution was remove this component

Reply:
Fixed

Reply:
thanks